tests/group_vars/vagrant.yml

```diff
@@ -45,6 +45,10 @@ dataverse:
     upload_to_hub: False
     user: counter
     year_month: "2018-05"
+  custom_metadata_blocks:
+    enabled: true
```

Comment:
vagrant.yml affects the tests on https://jenkins.dataverse.org right? We should probably disable it here too.